tensorboard/components/tensor_widget/demo/demo.html

```diff
@@ -0,0 +1,45 @@
+<!--
+@license
+Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>TensorWidget Demo</title>
+
+    <link rel="stylesheet" type="text/css" href="demo/demo.css" />
+  </head>
+
+  <body>
+    <h1>TensorWidget Demo</h1>
+    <p>
+      TensorWidget version:
+      <span id="tensor-widget-version"></span>
+    </p>
+
+    <script src="bundle.js"></script>
+    <script>
+      const TENSOR_WIDGET_MODULE_PATH =
+        'org_tensorflow_tensorboard/tensorboard/components/tensor_widget/tensor-widget';
```

Comment:
How is this string generated? Is it brittle (prone to breakage when files move around)? If so, can we write a (options):
1. write a test that checks, or
2. make this a template and use genrule to insert this path, maybe?

Reply:
As per discussion offline, we will add a test or genrule in subsequent PRs.

Reply:
Ack.